providers/implementations/kem/ecx_kem.c

```diff
@@ -363,16 +367,18 @@ int ossl_ecx_dhkem_derive_private(ECX_KEY *ecx, unsigned char *privout,
     if (kdfctx == NULL)
         return 0;
 
-    ossl_dhkem_getsuiteid(suiteid, kemid);
+    // ossl_dhkem_getsuiteid(suiteid, kemid);
+    suiteid[0] = kemid / 256;
```

Comment:
(kemid >> 8) &0xFF 
kemid & 0xFF
seems clearer..

Reply:
Happy either way but did that.